__tests__/CountValidator.test.js

```diff
@@ -0,0 +1,35 @@
+import CountValidator from '../src/utils/validator/CountValidator.js';
+import ErrorMessage from '../src/constants/ErrorMessage.js';
+
+test('시도횟수가 숫자가 아니면 에러 처리한다.', () => {
+  const TRY_COUNT = NaN;
+
+  expect(() => CountValidator.isNumber(TRY_COUNT)).toThrow(
+    ErrorMessage.isNotNumber,
+  );
+});
+
+// todo: test.each
+test('시도횟수가 0 이하이면 에러 처리한다.', () => {
+  const TRY_COUNT = 0;
+
+  expect(() => CountValidator.isAtLeastOne(TRY_COUNT)).toThrow(
+    ErrorMessage.isInvalidNumber,
+  );
+});
+
+test('시도횟수가 정수가 아니면 에러 처리한다.', () => {
+  const TRY_COUNT = 0.5;
+
+  expect(() => CountValidator.isInteger(TRY_COUNT)).toThrow(
+    ErrorMessage.isNotInteger,
+  );
+});
```

Comment:
각각의 에러 처리가 세분화 되어 있는만큼 테스트에서도 그게 드러나면 좋을거 같아요
```suggestion
test(`시도횟수가 정수가 아닌 경우 ${ErrorMessage.isNotInteger} 메세지가 포함된 에러를 throw한다.`, () => {
  const TRY_COUNT = 0.5;

  expect(() => CountValidator.isInteger(TRY_COUNT)).toThrow(
    ErrorMessage.isNotInteger,
  );
});
```